src/styles/main.scss

```diff
@@ -3,12 +3,28 @@
 
 #root {
     @import "containers/home";
+    background: $light-sea-green;
+}
+
+#outer-container {
```

Comment:
BEM yes. Somewhere else? Maybe. These are menu specific styles, but the elements are higher level. It was somewhat tricky, so I bottled the decision and just put the menu specific styles here, and anything not menu specific in the relevant higher level stylesheet.